tests/test_clos/test_process_workflow_list.py

```diff
@@ -0,0 +1,30 @@
+import requests, json
```

Comment:
```suggestion
import requests
import json
```
Please, use one import per line